### ISSUE TITLE
Greetings. I'm missing git and python, as well as Sonarr, Radarr, & Jackett. Looking for guidance.

### ISSUE THREAD

**Issue by BetaLeaf** [SynoCommunity/spksrc#3051]
Greetings. My name is Jeff Savage, or as I'm known online, BetaLeaf. I am new to Synology. I used to run a FreeNAS and Ubuntu server, so I do have some experience in BSD and Linux, including how to ssh into my box and editing text files. I still have a lot to learn.

-----

System: DS3617xs
Version: DSM 6.1.4-15217 Update 3

-----

 I have ran into an issue when trying to install the following programs:

CouchPotato & Headphones

When I click the install button, I receive this error:
```
The following packages are missing in the package server. Please contact the Synology support team for help. 

python, git
```
These packages do exist in package center, however, despite them being installed, I still cannot install CouchPotato and Headphones

-----

In addition to the above issue, I am also looking to install Radarr, Sonarr, and Jackett. I would prefer to use Radarr over CouchPotato. Unfortunately, these packages seem to be missing from package center and cannot be installed.

-----

In short, I am looking for guidance from a more experienced member of this community. I would greatly appreciate any advice given to me. Thank you for your time.

**Comment by ymartin59** [comment]
We are working hard to publish packages for DSM 6... we are probably closed to achievement but often faced complex issues.

**Comment by BetaLeaf** [comment]
Great to here, @ymartin59. Thanks for taking the time to reply. I believe I shall wait until support is available. Again, thank you for your time.
